pkg/thanos/rules.go

```diff
@@ -75,7 +75,7 @@ func (o *Operator) createOrUpdateRuleConfigMaps(t *monitoringv1.ThanosRuler) ([]
 		level.Debug(o.logger).Log(
 			"msg", "no PrometheusRule changes",
 			"namespace", t.Namespace,
-			"prometheus", t.Name,
+			"thanos", t.Name,
```

Comment:
Not sure, but which one is better `thanos` or `thanos ruler`?

Reply:
I think either is fine here.  I've been trying to use `thanos` in places where we could potentially re-use logic in the future for additional thanos components, and `thanos-ruler` when the code is specific just to thanos-ruler.